confd/conf.d/opennms.properties.d/ipc.properties.toml

```diff
@@ -0,0 +1,14 @@
+[template]
+src = "opennms.properties.d/ipc.properties.tpl"
```

Comment:
Why this file with RTC related features is called `ipc.properties` ? Is this a typo and should be named `rpc.properties` instead ? If this makes sense, why having `map-client` related features here ?